src/pages/locations/[id].tsx

```diff
@@ -0,0 +1,100 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { ErrorResponse } from "@/types";
+import { Location } from "@prisma/client";
+import {
+  LocationSignature,
+  getLocationSignature,
+} from "@/lib/client/localStorage/locationSignatures";
+
+const LocationDetails = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const [location, setLocation] = useState<Location>();
+  const [locationSignature, setLocationSignature] =
+    useState<LocationSignature>();
+
+  useEffect(() => {
+    const fetchLocation = async () => {
+      if (typeof id === "string") {
+        try {
+          const response = await fetch(`/api/location/${id}`);
```

Comment:
Any reason to protect this endpoint with a ZK PoK of signature? like are any locations gonna be private on purpose for scavenger hunts? Def doesn't need to happen in this PR but if you think it's important we should add an issue

Reply:
There could be cases where this is useful, such as a scavenger hunt - i think in that case we can always protect the individual location with a PoK. If we plan on having such an activation its worth keeping in mind but for now doesn't seem to warrant the issue imo